tests/foreman/cli/test_oscap.py

```diff
@@ -1127,6 +1128,48 @@ def test_positive_delete_scap_policy_with_name(self):
         with self.assertRaises(CLIReturnCodeError):
             Scapcontent.info({'name': scap_policy['name']})
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_associate_scap_policy_with_single_server(self):
+        """Assign an audit policy to a single server
+
+        :id: 30566c27-f466-4b4d-beaf-0a5bfda98b89
+
+        :setup:
+
+            1. Oscap should be enabled.
+            2. Oscap-cli hammer plugin installed.
+            3. At least 1 policy and host.
+
+        :steps:
+
+            1. Login to hammer shell.
+            2. Execute "policy" command with "update" as sub-command.
+            3. Pass host name as the parameter.
+
+        :expectedresults: The scap policy is updated.
+        """
+        host = entities.Host()
+        host.create()
+        name = gen_string('alpha')
+        scap_policy = make_scap_policy({
+            'name': name,
+            'scap-content-id': self.scap_id_rhel6,
+            'scap-content-profile-id': self.scap_profile_id_rhel6,
+            'period': OSCAP_PERIOD['weekly'].lower(),
+            'weekday': OSCAP_WEEKDAY['friday'].lower()
+        })
+        host_name = host.name + "." + host.domain.name
+        Scappolicy.update({
+            'id': scap_policy['id'],
+            'hosts': host_name,
+        })
+        data = entities.Policies(id=scap_policy['id']).read().hosts
```

Comment:
as we are in hammer testing and there is a way to retrieve the policies hosts, prefer to use hammer command over API calls
```
hammer host list --search compliance_policy = POLICY_NAME
```
This will return the policy hosts
eg:
```python
from robottelo.cli.host import Host

hosts = Host.list({'search': 'compliance_policy = {0}'.format(name)})
```

Reply:
compliance_policy_id also can be used
```python
hosts = Host.list({'search': 'compliance_policy_id = {0}'.format(scap_policy['id'])})
```

Reply:
I have asked Silvie and this not core of the test, this is just a check that `Scappolicy.update(...)` succeeded - and that comes from `from robottelo.cli.scap_policy import Scappolicy` so we should be OK.

Reply:
Think that this is part of the test as we are relying on this to ensure that  "The scap policy is updated."
Think if we put it as CLI cmd , this will bring more value to the test it self as not only we going to assert but ensure also that the compliance policy hosts list is working on the CLI also.